crates/payload/basic/src/lib.rs

```diff
@@ -679,6 +682,28 @@ where
         // convert tx to a signed transaction
         let tx = pool_tx.to_recovered_transaction();
 
+        if let Transaction::Eip4844(blob_tx) = &tx.transaction {
+            // only process this tx if cancun is active
+            if chain_spec.is_cancun_activated_at_timestamp(attributes.timestamp) {
+                // mark tx and all dependent txs as invalid because we can't process blob txs
+                // until cancun is active
+                best_txs.mark_invalid(&pool_tx);
+                continue
+            }
```

Comment:
@mattsse do fork checks in the txpool make this redundant?

Reply:
yeah, they're only accepted when the timestamp has been reached, this also does not affect reorgs, so we can omit the check here.

Reply:
cool, removed